assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convertors.py

```diff
@@ -836,18 +935,20 @@ def save_as_mlflow(self):
         """Prepare model for save to MLflow."""
         from azureml.automl.dnn.vision.common.mlflow.mlflow_model_wrapper import MLFlowImagesModelWrapper
         from azureml.automl.dnn.vision.common.model_export_utils import _get_scoring_method
-
-        mlflow_model_wrapper = MLFlowImagesModelWrapper(
-            model_settings={},
-            task_type=self._task,
-            scoring_method=_get_scoring_method(self._task),
-        )
+        is_yolo = self._task == SupportedTasks.IMAGE_OBJECT_DETECTION.value
+        mlflow_model_wrapper = MLFlowImagesModelWrapper(model_settings={},
```

Comment:
We have some json file in most of the model cards, which is read and then used to do logging. This is then used to track usage of this model. Madhu worked on it. Check once with Madhu if logging would happen when this model card is used.